docs/submitter.rst

```diff
@@ -118,9 +120,10 @@ We follow the steps of the OpenID Connect protocol.
 - The OP responds with an ID Token and usually an Access Token, which are validated with configuration provided by ``OIDC_URL``.
 - The RP can send a request with the Access Token to the UserInfo Endpoint.
 - The UserInfo Endpoint returns Claims about the End-User, use claims ``sub``, ``CSCUserName`` or ``remoteUserIdentifier`` to identify the user and start a session.
+- If using LifeScience AAI as authentication provider, then ``AAI_LIFESCIENCE_ROOT_GROUP`` must be set. This configuration is not used with CSC AAI.
```

Comment:
Does it mean that we can only use one or the other?

Reply:
You can set this also if you're using CSC AAI, but it does nothing. The authentication source is controlled by `OIDC_URL` and there we can only have one authentication source. This new env is only used for parsing the very long strings from LS into a shorter format.

Reply:
SDS AAI also has LS support. Isn't that enough for us? So we use only SDS AAI?